### ISSUE TITLE
Const values precede dataclass definitions in output

### ISSUE THREAD

**Issue by soaxelbrooke** [unmade/thrift-pyi#47]
The following thrift:

```thrift
namespace scala test

struct Foo {
    1: required string bar
    2: required i64 baz
}

const Foo foo = {
    "bar": "value",
    "baz": 1234
}
```

Turns into the following python:

```python
from dataclasses import dataclass
from typing import *

foo: Any = Foo(bar="value", baz=1234)

@dataclass
class Foo:
    bar: str
    baz: int

```

Which results in the following exception when importing:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/stuart/code/thriftdemo/thriftpy/test.py", line 4, in <module>
    foo: Any = Foo(bar="value", baz=1234)
               ^^^
NameError: name 'Foo' is not defined
```

**Comment by unmade** [comment]
Thank you for reporting the issue! I'll try to give it a look on weekends

**Comment by soaxelbrooke** [comment]
Wow, thank you for the quick fix!

**Comment by unmade** [comment]
released in version 0.9.0
